bigquery_etl/glam/templates/clients_histogram_aggregates_v1.init.sql

```diff
@@ -10,7 +10,6 @@ CREATE TABLE IF NOT EXISTS
     channel STRING,
     histogram_aggregates ARRAY<
       STRUCT<
-        latest_version INT64,
```

Comment:
This field causes issues when not using `latest_versions` to filter app versions. This doesn't apply on nightly because app versions are just build versions. 

I double checked the desktop aggregates, and it looks like the bucket count query doesn't reference latest versions at all, unlike the query in this module: https://github.com/mozilla/bigquery-etl/blob/master/sql/telemetry_derived/clients_histogram_bucket_counts_v1/query.sql

While it is being used in the histogram aggregates for filtering against the new data in the desktop queries, it is mistakenly being aggregated in the bucket counts here. I'm removing it since it doesn't need to be in the query.